src/lib/common/sem.h

```diff
@@ -37,15 +37,15 @@
 
 /* ****************************************************************************
 *
-* SemRequestType - 
+* SemOpType -
 */
-typedef enum SemRequestType
+typedef enum SemOpType
```

Comment:
I changed this to become more general... to discover at the end that cache sem were not using this :(

Should I rollback the change? Or the "SemOpType" is also a good name? (It is shorter than the former)